tests/test_mcts.py

```diff
@@ -177,6 +221,9 @@ def test_never_select_illegal_moves(self):
             self.assertNotEqual(leaf.fmove, 1)
 
     def test_dont_pick_unexpanded_child(self):
+        # CPUCT must be large enough that it can overcome Q = -1/2
```

Comment:
I don't understand this change - was this test failing after your changes?

Reply:
Yes, previously puct was magnified (`sqrt(N + 3)` is much larger than `sqrt(N)` for small Ns) and now this test fails.